Makefile

```diff
@@ -73,7 +73,7 @@ fmt:
 
 # Run go vet against code
 vet:
```

Comment:
Should we add it to the `go fmt` make recipe?

Reply:
in go 1.13 fmt throws an error - https://github.com/golang/go/issues/27841

if we don't care about supporting 1.13, i think we can add it yes.

Reply:
Ah, ok. We can hold off adding it as I've updated the go version dependency in `go.mod` in some of the installation work.